.github/dependabot.yml

```diff
@@ -136,12 +136,14 @@ updates:
           - com.google.firebase:firebase-crashlytics-ndk
           - com.google.firebase:firebase-analytics-ktx
 
+# FIXME: Bring back regular swift defaults (weekly, 28 day cooldown)
```

Comment:
You shouldn't need this as any changes to dependabot's config will cause an immediate check.

You can also view dependabot runs in the "Insights" tab of the repo.

Reply:
update: I've just downgraded to 8.49 (the version used earlier). this means that dependabot insight should follow straight away.